notebooks/custom-prompts-demo.ipynb

```diff
@@ -0,0 +1,571 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "516c8a3a-a2dd-434c-876a-09d595e839d6",
+   "metadata": {},
+   "source": [
+    "<a href=\"https://colab.research.google.com/github/vectara/example-notebooks/blob/main/notebooks/custom-prompts-demo.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1cbe98ff-8464-4ee5-8b42-9242ae5f9455",
+   "metadata": {},
+   "source": [
+    "# Powerful features with Vectara Scale plan\n",
+    "\n",
+    "Vectara's [scale plan](https://vectara.com/pricing/) provides a lot of powerful features for building RAG pipelines. \n",
+    "In this notebook we will focus on two of the most powerful features:\n",
+    "1. The ability to use GPT-4 or GPT4-Turbo instead of GPT-3.5-Turbo.\n",
+    "2. Using custom prompts with Vectara\n",
+    "\n",
+    "Both of these features help get better responses from your RAG pipeline - more accurate responses, better summarization, and less hallucinations."
```

Comment:
I think the benefit statement applies to the first of these 2, but I've seen a number of users "mess up" custom prompts in ways that yield higher hallucinations.  The benefit for custom prompts is really around allowing complete control of the style and tone of the output